src/server_status.cc

```diff
@@ -0,0 +1,84 @@
+#include "server_status.h"
+
+#include <string>
+#include <vector>
+#include <map>
+
+namespace http{
+namespace server{
+
+
+
+long long
+ServerStatus::getRecordNum(){
+	return total_visit;
+}
+
+std::string
+ServerStatus::responseToString(ResponseCode response){
+	switch(response){
+		case  ResponseCode::ok:
+		return "200";
+		case  ResponseCode::bad_request:
+		return "400";
+		case ResponseCode::not_found:
+		return "404";
+		default:
+		return "500";
+	}
+}
+
+void
+ServerStatus::addHandlerToUri(std::string handlerName, std::string uri){
+	handler_to_uri[handlerName].push_back(uri);
+}
+
+void 
+ServerStatus::addRecord(std::string uri, ResponseCode response){
+	if(uri_visit_count.count(uri) <= 0)
```

Comment:
That's a good idea considering the counting should be done in different places. Let me split them.